### PR TITLE
small change to two_a_array_ll to use mock

### PR DESCRIPTION
**Pull request by M-iceberg**

### CODE REVIEW

File: inc/c_util/two_d_array_ll.h [hunk 1]
```diff
@@ -259,4 +259,21 @@ all_ok:
     return result;                                                                                                                                         \
 }                                                                                                                                                          \
 
+/*macro to be used in headers*/                                                                                     \
```
Comment (darobs) [inline inc/c_util/two_d_array_ll.h:262]: To be used in which headers?

**Review by darobs — APPROVED** [review]
:shipit:

File: inc/c_util/two_d_array_ll.h (hunk 1 above)
Comment (M-iceberg) [reply inc/c_util/two_d_array_ll.h:262]: will be used in reals

**Review by darobs — APPROVED** [review]
:shipit:

**Review by dcristoloveanu — APPROVED** [review]
:shipit:

**Pull request merged**